tests/unit/test_store_synapse.py

```diff
@@ -57,3 +60,55 @@ def test_get_file_entityIds_all_files(self) -> None:
             "Filename": ["test_file_1.txt", "test_file_2.txt"],
             "entityId": ["syn2", "syn3"],
         }
+
+    @pytest.mark.parametrize(
```

Comment:
Could you add a case here where the input df has both "Id" and "Uuid", and the output is unchanged?

Reply:
Could you also add a case where the input df has neither `Id` or `entityId` columns and the output has both?

Reply:
These tests were added. But I want to note that the conflict resolution between Id and uuid was not previously present. Since it is not difficult to add, I also added it in the PR (but this is also probably outside of the initial scope)